pkg/cli/api.go

```diff
@@ -60,8 +60,8 @@ func (c cli) bindCreateAPI(ctx plugin.Context, cmd *cobra.Command) {
 		tmpGetter, isGetter := p.(plugin.CreateAPIPluginGetter)
 		if isGetter {
 			if getter != nil {
-				err := fmt.Errorf("duplicate API creation plugins for project version %q: %s, %s",
```

Comment:
IMO this error should not occur at all. Note that `c.resolvedPlugins` will return all supported plugins for the PROJECT version which can be, for example: `go.kubebuilder.io/v3.0.0, go.kubebuilder.io/v3.1.0, go.kubebuilder.io/v3.2.0`

Then, we should use the upper version in this scenario as described in https://github.com/kubernetes-sigs/kubebuilder/issues/1526.

Note that this problem has no longer been faced because KB has just 1 plugin version at the moment which is implemented.